src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/LanguageServerEndpointAttribute.cs

```diff
@@ -17,8 +17,14 @@ public class LanguageServerEndpointAttribute : Attribute
     /// </summary>
     public string Method { get; }
 
-    public LanguageServerEndpointAttribute(string method)
+    /// <summary>
+    /// Contains any language names that this <see cref="IMethodHandler"/> is specific to.
+    /// </summary>
+    public string[] Languages { get; }
```

Comment:
```suggestion
    public ImmutableArray<string> Languages { get; }
```

Reply:
nit: while you're here, can you seal this type?

Reply:
We can't seal it. MethodAttribute and XamlMethodAttribute derive from it.